Detectors/TPC/workflow/src/ClusterConverterSpec.cxx

```diff
@@ -44,15 +44,15 @@ DataProcessorSpec getClusterConverterSpec()
     auto processingFct = [](ProcessingContext& pc) {
       // this will return a span of TPC clusters
       auto inClusters = pc.inputs().get<std::vector<o2::TPC::Cluster>>("clusterin");
-      int nClusters = inClusters.size();
+      int nClusters = inClusters->size();
       LOG(INFO) << "got clusters from input: " << nClusters;
 
       std::vector<ClusterHardwareContainer> containerMetrics;
       unsigned clusterIndex = 0;
       for (; clusterIndex < nClusters; clusterIndex++) {
         // clusters are supposed to be sorted in CRU number, start a new entry
         // for every new CRU, a map is needed if the clusters are not ordered
-        auto inputcluster = inClusters[clusterIndex];
+        auto inputcluster = (*inClusters)[clusterIndex];
```

Comment:
This is why I actually do not like it.. I expect that retrieving containers is by far the most common usecase, compared to retrieving a single object. Is that not the case?